vault/resource_database_secret_backend_connection_test.go

```diff
@@ -19,6 +22,12 @@ import (
 	"github.com/hashicorp/terraform-provider-vault/testutil"
 )
 
+const testDefaultDatabaseSecretBackendResource = "vault_database_secret_backend_connection.test"
+
+// TODO: add support for automating tests for plugin_name
+// Currently we have to configure the Vault server with a plugin_directory,
+// copy/build a db plugin and install it with a unique name, then register it in vault.
+
```

Comment:
@vinay-gopalan, in regards to https://github.com/hashicorp/terraform-provider-vault/pull/1320#discussion_r794094203 - do you think this sufficient for now, or would you rather I document the procedure somewhere else?

Reply:
Ah I think I now understand what you meant by this comment. I didn't fully take into account the Vault server setup required for custom plugin names. I believe this comment is sufficient, and resolves my query